src/analysis/normed_space/basic.lean

```diff
@@ -414,6 +438,27 @@ lemma norm_pow_le {α : Type*} [normed_ring α] (a : α) : ∀ {n : ℕ}, 0 < n
            (mul_le_mul (le_refl _)
                        (norm_pow_le (nat.succ_pos _)) (norm_nonneg _) (norm_nonneg _))
 
+lemma norm_pow_le_eventually {α : Type*} [normed_ring α] (a : α) :
```

Comment:
```suggestion
lemma eventually_norm_pow_le {α : Type*} [normed_ring α] (a : α) :
```
BTW, https://mathworld.wolfram.com/NormedRing.html requires `norm 1 = 1`. Should we do the same (not for this PR)?

Reply:
My instinct would be to wait until there is a theorem that needs that hypothesis.  At the moment there are very few instances of `normed_ring` (all of which do have this property), so it will not be hard to make the change and fix all the instances if needed.